app/helpers/pipeline_runs_helper.rb

```diff
@@ -109,11 +109,11 @@ def fetch_pipeline_version(s3_file = pipeline_version_file)
     Regexp.last_match(1)
   end
 
-  def upload_dag_json_and_return_job_command(dag_json, dag_s3, dag_name, key_s3_params = nil, copy_done_file = "")
```

Comment:
I split this method into two: `upload_dag_json` and `dag_job_commands`, since there doesn't seem to be a reason why they are combined in one method (other than sharing some variables)